### PR TITLE
[DOCS] Refine docs about EA > Kafka > LS > ES

### PR DESCRIPTION
**Pull request by lucabelluccini**
Update the statements related to EA > Kafka > LS > ES to remove the mention of testing and make it "supported"

### CODE REVIEW

**Comment by lucabelluccini** [comment]
Sorry for editing again those statements but we had some follow up discussions and with @flexitrev we agreed we should remove the statement and make this flow supported.

I strongly advise to ask Trevor for a final review of the docs prior to merging & backporting.

File: docs/en/ingest-management/fleet/fleet-settings-output-kafka.asciidoc [hunk 1]
```diff
@@ -8,7 +8,12 @@ Specify these settings to send data over a secure connection to Kafka. In the {f
 == Kafka output and using {ls} to index data to {es}
 
 If you are considering using {ls} to ship the data from `kafka` to {es}, please
-be aware Elastic is not currently testing this kind of setup.
+be aware the structure of the documents sent from {agent} to `kafka` must not be modified by {ls}.
+We suggest disabling `ecs_compatibility` on both the `kafka` input and the `json` codec in order
+to make sure the input doesn't edit the fields and their contents.
+
+The data streams setup by the integrations expect to receive events having the same structure and
```
Comment (kilfoyle) [inline docs/en/ingest-management/fleet/fleet-settings-output-kafka.asciidoc:15]:
```suggestion
The data streams set up by the integrations expect to receive events having the same structure and
```

File: docs/en/ingest-management/elastic-agent/configuration/outputs/output-kafka.asciidoc [hunk 2]
```diff
@@ -46,11 +46,13 @@ outputs:
...
-If you are considering using {ls} to ship the data from `kafka` to {es}, please
-be aware Elastic is not currently testing this kind of setup.
+If you are considering using {ls} to ship the data from `kafka` to {es}, please be aware the 
+structure of the documents sent from {agent} to `kafka` must not be modified by {ls}.
+We suggest disabling `ecs_compatibility` on both the `kafka` input and the `json` codec in order
+to make sure the input doesn't edit the fields and their contents.
 
-The structure of the documents sent from {agent} to `kafka` must not be modified by {ls}.
-We suggest disabling `ecs_compatibility` on both the `kafka` input and the `json` codec.
+The data streams setup by the integrations expect to receive events having the same structure and
```
Comment (kilfoyle) [inline docs/en/ingest-management/elastic-agent/configuration/outputs/output-kafka.asciidoc:54]:
```suggestion
The data streams set up by the integrations expect to receive events having the same structure and
```

**Review by kilfoyle — APPROVED** [review]
LGTM! 🚀 

Thanks for fixing this @lucabelluccini. Once CI checks pass I'll merge and backport it into the current docs.

**Review by karenzone — APPROVED** [review]
LGTM 🚀

**Pull request merged**